wgpu-hal/src/vulkan/adapter.rs

```diff
@@ -1919,6 +1919,7 @@ impl super::Adapter {
                 } else {
                     spv::ZeroInitializeWorkgroupMemoryMode::Polyfill
                 },
+                force_loop_bounding: true,
```

Comment:
Can we get some way to turn this off? My program has some very performance sensitive small loops.

Reply:
This should be wired up to the switch already in the hal shader module descriptor.

Reply:
Is it exposed to naga/wgpu for end users? I'm not seeing it, maybe I missed it though.

Reply:
Yes - Device::create_shader_module_trusted lets you turn off various protections against ub.

Reply:
Ah thank you!

Reply:
Thanks for raising this, I've updated the commit message to clarify how to disable it.

Reply:
nit: the commit messages are basically ephemeral, this should be in the changelog, preferably in the major changes section